vuepress/docs/next/docs/create/component-gen-overview.md

```diff
@@ -34,11 +34,11 @@ This section provides an overview of the component generation process. If you wa
 2. Define bundle and deploy assets
    - The Entando blueprint includes scripts and tools to automatically generate a bundle ready for deployment to the ECR. The bundle will include your microservice, generated microfrontends, and any microfrontends that you have created manually under the `ui` folder
    - The generated Spring Boot microservice can be built and deployed to a Docker registry using the provided maven Jib plugin
-   - Generated bundles can be pushed to a git repository of the users choice
+   - Generated bundles can be pushed to a Git repository of the users choice
```

Comment:
docker/v5? TODO or fix now???

Reply:
@nshaw @jyunmitch TODO... all of the details on this page are specific to the jhipster generate ms and mfe as it is now; we'd either want to discuss/link to v1 -> v5 conversion or rewrite for v5 generation (assuming that comes to fruition)

Reply:
@Lyd1aCla1r3 Let's get a new ticket. I think we want to rewrite for v5. I'm requesting updating the blueprint to v5 for 7.1 (https://entando.myjetbrains.com/youtrack/issue/ECS-463/Convert-the-jhipster-blueprint-to-the-new-DockerOCIv5-format)